pkg/api/profile/profile_handler.go

```diff
@@ -0,0 +1,81 @@
+package profile
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/shirrashko/BuildingAServer-step2/pkg/api/model"
+	"github.com/shirrashko/BuildingAServer-step2/pkg/bl/profile"
+	"net/http"
+	"strconv"
+)
+
+type Handler struct {
+	service *profile.Service
+}
+
+func NewHandler(s *profile.Service) Handler {
+	return Handler{s}
+}
+
+func (h Handler) getProfileByID(c *gin.Context) {
+	// Retrieve the id path parameter from the URL
+	id, err := strconv.Atoi(c.Param("id"))
+	if err != nil {
+		c.IndentedJSON(http.StatusBadRequest, gin.H{"message": "Invalid ID"})
+		return
+	}
+
+	// Check if the user exists in the database
+	if !h.service.IsUserInDB(id) {
+		c.IndentedJSON(http.StatusNotFound, gin.H{"message": "User not found"})
```

Comment:
c.IndentedJSON also sends a JSON response with "Content-Type: application/json," but it additionally applies indentation to the JSON data to make it more human-readable. This can be useful for debugging purposes or when you want to inspect the response directly in the browser or other tools. But when  you use a tool like Postman to make requests and view responses, the difference between c.JSON and c.IndentedJSON might not be immediately apparent. Is it better that i will use c.JSON?